.browserslistrc

```diff
@@ -0,0 +1,18 @@
+[production]
+>0.5% in alt-eu and >0.5% in alt-as and >0.5% in alt-na and >0.5% in alt-sa and >0.5% in alt-af and >0.5% in alt-oc and >1%
+not ie > 0
+last 2 Chrome versions
+last 2 Edge versions
+last 2 Safari versions
+Firefox esr
+
+[supported]
+last 2 Chrome versions
+last 2 Firefox versions
+last 2 Edge versions
+last 2 Safari versions
+
+[development]
```

Comment:
isn't it risky to use different presets in dev/prod? nothing more annoying than issues just happening in prod builds

Reply:
OTOH, compile time would probably go down. But yeah, agreed.

Reply:
I'm just wondering whether I should replicate the list from 'production' or 'supported'.